test/e2e/clusterlogging_test.go

```diff
@@ -215,78 +202,37 @@ func clusterLoggingUpgradeClusterTest(t *testing.T, f *framework.Framework, ctx
 		{Name: "RSYSLOG_IMAGE", Value: "quay.io/viaq/rsyslog:upgraded"},
 	}
 
+	t.Logf("Modified image ENV variables to force upgrade: %q", newEnv)
 	currentOperator.Spec.Template.Spec.Containers[0].Env = newEnv
 	err = f.Client.Update(goctx.TODO(), currentOperator)
 	if err != nil {
 		return fmt.Errorf("could not update cluster-logging-operator with updated image values %v", err)
 	}
 
-	err = CheckForElasticsearchImageName(t, f.Client, namespace, "elasticsearch", "quay.io/openshift/origin-logging-elasticsearch5:upgraded", retryInterval, timeout)
+	err = CheckForElasticsearchImageName(t, f.Client, namespace, "elasticsearch", getValueFromEnvVar(newEnv, "ELASTICSEARCH_IMAGE"), retryInterval, timeout)
 	if err != nil {
 		return err
 	}
 
-	err = CheckForDeploymentImageName(t, f.KubeClient, namespace, "kibana", "quay.io/openshift/origin-logging-kibana5:upgraded", retryInterval, timeout)
+	err = CheckForDeploymentImageName(t, f.KubeClient, namespace, "kibana", getValueFromEnvVar(newEnv, "KIBANA_IMAGE"), retryInterval, timeout)
 	if err != nil {
 		return err
 	}
-
-	err = CheckForCronJobImageName(t, f.KubeClient, namespace, "curator", "quay.io/openshift/origin-logging-curator5:upgraded", retryInterval, timeout)
+	err = CheckForDeploymentImageName(t, f.KubeClient, namespace, "kibana", getValueFromEnvVar(newEnv, "OAUTH_PROXY_IMAGE"), retryInterval, timeout)
 	if err != nil {
 		return err
 	}
 
-	if collector == "rsyslog" {
-		err = CheckForDaemonSetImageName(t, f.KubeClient, namespace, collector, "quay.io/viaq/rsyslog:upgraded", retryInterval, timeout)
-		if err != nil {
-			return err
-		}
-	}
-	if collector == "fluentd" {
-		err = CheckForDaemonSetImageName(t, f.KubeClient, namespace, collector, "quay.io/openshift/origin-logging-fluentd:upgraded", retryInterval, timeout)
-		if err != nil {
-			return err
-		}
-	}
-
-	currentOperator, err = f.KubeClient.AppsV1().Deployments(namespace).Get("cluster-logging-operator", metav1.GetOptions{IncludeUninitialized: true})
-	if err != nil {
-		return fmt.Errorf("failed to get currentOperator: %v", err)
-	}
-
-	currentOperator.Spec.Template.Spec.Containers[0].Env = currentEnv
-	err = f.Client.Update(goctx.TODO(), currentOperator)
-	if err != nil {
-		return fmt.Errorf("could not update cluster-logging-operator with prior image values %v", err)
-	}
-
-	err = CheckForElasticsearchImageName(t, f.Client, namespace, "elasticsearch", "quay.io/openshift/origin-logging-elasticsearch5:latest", retryInterval, timeout)
```

Comment:
shouldn't we keep this?

Reply:
No.  This is what we talked about on slack.  This begins check where we modify the image back to the original which is re-testing the fact that the operator can react to image changes

Reply:
missed the check for it earlier...